tests/test-config.json

```diff
@@ -217,6 +218,14 @@
         "basePath should show 404 for page not under the /docs prefix"
       ]
     },
+    {
+      "file": "test/e2e/basepath/error-pages.test.ts",
+      "reason": "Hard-coded expectation of Vercel's default 404 page",
+      "tests": [
+        "basePath should not update URL for a 404",
+        "basePath should show 404 for page not under the /docs prefix"
+      ]
```

Comment:
there were three tests skipped here previously, but one of these is now passing so I didn't include it